### PR TITLE
update README with "Automatic code reloading" instructions

### PR DESCRIPTION
**Pull request by diogovk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,16 @@
 
 Now you can visit `localhost:4000` from your browser.
 
+## Automatic code reloading
+
+1. Install the `nodejs package` (or your system's equivalent) if you don't have it already
+1. Install npm dependencies with `npm install`
+1. Install `inotify-tools` with the package manager of your preference:
```
Comment (marcioj) [inline README.md]: I don't remember to install inotify-tools is it something specific of Arch linux?

**Comment by diogovk** [comment]
I'm pretty sure you had it installed already.
Here is the problem I was having before installing inotify-tools:
https://github.com/phoenixframework/phoenix/issues/776
And here the install guide:
https://github.com/rvoicilas/inotify-tools/wiki

**Comment by diogovk** [comment]
@marcioj Also, inotify-tools is mentioned in the official docs:
http://www.phoenixframework.org/v0.12.0/docs/overview

**Comment by marcioj** [comment]
Thanks @diogovk

**Comment by sanford** [comment]
Having just hit the issue on my linux install, seems the inotify instructions should also be on the installation page, not just the overview page:

http://www.phoenixframework.org/docs/installation

**Comment by diogovk** [comment]
@sanford Actually I've just checked and there's a mention in the bottom of the page that Linux users should install inotify.

**Comment by sanford** [comment]
My bad.  Missed it completely 2x.  Perhaps we could catch the error and show a suggested install message on Linux?

**Comment by diogovk** [comment]
You should suggest this in the phoenix project then.

**Pull request merged**